src/index.tsx

```diff
@@ -37,7 +37,7 @@ if (window.top === window.self) {
 }
 
 const archiveUrl = config.REACT_APP_ARCHIVE_URL_OVERRIDE ||
-  assertDefined(config.REACT_APP_ARCHIVE_URL, 'REACT_APP_ARCHIVE_URL must be defined');
+  assertDefined(getArchiveUrlSync(), 'archive url must be defined');
```

Comment:
when this changes later it won't be used, something else needs to be changed to make this work